### PR TITLE
Jerren dg update

### PR DESCRIPTION
**Pull request by jerrrren**
Update user stories 
fix toc

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -215,7 +214,7 @@ Sequence Diagram:
 
 #### Flow
 1. The user enters the command, eg. `add id/37 n/John p/9861 7251 e/John@nus.com a/nus t/Vegetarian`
-2. The parser will parse the argument and instantiate several fields: Id, name, phone, email, address, and tags, respective with the prefixes.
+2. The parser will parse the argument and in several fields: Id, name, phone, email, address, and tags, respectively with the prefixes.
```
Comment (4ndrelim) [inline docs/DeveloperGuide.md:217]: "instantiate" is intended here. Parser actually creates these objects

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -239,7 +238,7 @@ Sequence Diagram:
 
 #### Flow
 1. The user enters the command, eg. `edit eid/37 p/8461 4872 a/ntu`. The employee with Id 37 will be identified and will have its phone and address fields updated as specified, if it exists in SudoHR.
-2. The parser will instantiate a new `Phone` and `Address` object constructed from the arguments associated with `/p` and `/n`, which represent the new phone number and address fields respectively.
+2. The parser will instantiates a new `Phone` and `Address` object constructed from the arguments associated with `/p` and `/n`, which represents the new phone number and address fields respectively.
```
Comment (4ndrelim) [inline docs/DeveloperGuide.md:241]: thx for checking grammer but there no 's' for the verb that comes after the word 'will'

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -267,15 +266,14 @@ Sequence Diagram:
 
 #### Flow
 1. The user enters the command, eg. `del eid/37` where employee with Id 37 is to be removed from SudoHR.
-2. The parser will instantiate the corresponding `Id` object constructed from the argument associated with the prefix `eid/`.
+2. The parser will instantiates the corresponding `Id` object constructed from the argument associated with the prefix `eid/`.
```
Comment (4ndrelim) [inline docs/DeveloperGuide.md:269]: here as well

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -538,12 +539,13 @@ Sequence Diagram:
 ##### Flow
 
 1. The user enters the command `aetl eid/1 d/2022-03-04` where 1 is the employee id and 2022-03-04 is the leave date.
-2. The parser would initialise a new `Id` and `LeaveDate` object constructed from the input of argument `eid/` and `d/` respectively
+2. The parser would i a new `Id` and `LeaveDate` object constructed from the input of argument `eid/` and `d/` respectively
```
Comment (4ndrelim) [inline docs/DeveloperGuide.md:542]: typo 'i' here?

**Review by 4ndrelim — CHANGES_REQUESTED** [review]

**Review by kennethk-1201 — APPROVED** [review]
lgtm

**Pull request merged**